### ISSUE TITLE
[fr] android joy

### ISSUE THREAD

**Issue by griffindoors** [cryptag/cryptag#94]
Will there be android joy? possibly F-Droid binaries?

**Comment by elimisteve** [comment]
Yes!  The first CrypTag-based Android app will either be an app for handling arbitrary text, or a basic version of [CrypTag Notes](https://github.com/cryptag/notes/issues/49).
